src/main.js

```diff
@@ -739,6 +739,15 @@ function initializeAfterAppReady() {
       defaultPath: os.homedir() + '/Downloads/' + filename,
       filters,
     });
+
+    item.on('done', (doneEvent, state) => {
+      if (state === 'completed') {
+        mainWindow.webContents.send('download-complete', {
+          title: filename,
+          defaultPath: os.homedir() + '/Downloads/' + filename,
+        });
```

Comment:
should we have any notification if state was other than completed?

Reply:
Probably a good idea to have error handling but not sure how to test it.